app/src/main/java/com/kickstarter/libs/utils/Secrets.java.example

```diff
@@ -77,8 +77,7 @@ public final class Secrets {
   }
 
   public static final class Privacy {
-    public static final String DELETE_ACCOUNT = "https://kickstarter.privacy";
-    public static final String REQUEST_DATA = "https://www.kickstarter.privacy";
+    public static final String TRANSCEND_PRIVACY_REQUEST_FLOW = "https://legal.kickstarter.com/policies/en/?modal=take-control";
```

Comment:
Should this be a dummy string? Or is it ok to put the real url here

Reply:
For this concrete case it's ok to put the real URL, it is a public accessible url, but as rule of thumbs for the anything coming from native-secrets will use dummy url